packages/core/src/components/_index.scss

```diff
@@ -30,17 +24,18 @@ Styleguide components.usage
 /*
 NPM installation
 
-Run `npm install`, which will pull in dependencies for you:
+Install the core package with an NPM client like `npm` or `yarn`, pulling in all relevant dependencies:
 
 ```sh
 npm install --save @blueprintjs/core
 ```
 
-Finally, you'll need to include the global type definitions for React and ReactDOM in your project using
-[Typings][typings] or [TSD][tsd]. Blueprint's own typings, `blueprint.d.ts`, are distributed in the NPM module and
-should be resolved automatically by the [compiler][tscompiler].
+If you are using TypeScript, you'll need to include ambient type definitions for React and ReactDOM in your
```

Comment:
would be cool to have a complete list of the minimal set of typings necessary. that should be considered part of installing blueprint.